packages/devtools_app/lib/src/shared/diagnostics/diagnostics_node.dart

```diff
@@ -8,7 +8,6 @@ import 'package:collection/collection.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
-import 'package:vm_service/vm_service.dart';
```

Comment:
I recommend that we remove the changes to this file. This class is extended and used by a customer inside google, so we do not want to risk breaking them. Also, we some of these helpers will be useful as we continue to evolve the Inspector and Layout Explorer and add advanced functionality.

Reply:
Sure, done.